### PR TITLE
AWS IoT plugin

### PR DESCRIPTION
**Pull request by ted-gould**
Copying over from LP

### CODE REVIEW

**Comment by sergiusens** [comment]
Can you get `master` rebased/merged to get the .travis.yml?

**Comment by ted-gould** [comment]
I think I attach an issue like this: #33

**Comment by ted-gould** [comment]
And #27

File: snapcraft/plugins/python2.py [hunk 1]
```diff
@@ -48,6 +48,15 @@ def schema(cls):
         schema['properties']['requirements'] = {
             'type': 'string',
         }
+        schema['properties']['pip-packages'] = {
```
Comment (sergiusens) [inline snapcraft/plugins/python2.py]: Can you rename this to `python-packages`? `pip` is just the tool we use to reach the *python package index*

File: snapcraft/plugins/python3.py [hunk 2]
```diff
@@ -48,6 +48,15 @@ def schema(cls):
         schema['properties']['requirements'] = {
             'type': 'string',
         }
+        schema['properties']['pip-packages'] = {
```
Comment (sergiusens) [inline snapcraft/plugins/python3.py]: same

**Comment by ted-gould** [comment]
Oh, don't forget #8

**Comment by sergiusens** [comment]
Can you try and merge master here to see if everything still works?

**Pull request merged**